techlibs/gatemate/synth_gatemate.cc

```diff
@@ -301,6 +301,7 @@ struct SynthGateMatePass : public ScriptPass
 			}
 			run("muxcover " + muxcover_args);
 			run("opt -full");
+			run("simplemap");

```

Comment:
`simplemap` will break down `$mux` into individual `$_MUX_` slices so I would expect it sequenced before `muxcover` which expects `$_MUX_` cells for its input. Then there's `simplemap` in the `map_regs` section already ahead of `muxcover`

Reply:
Problem here is that in this case after `opt -full` `$mux` cells would appear again, so `simplemap` is needed them to get them back to `$_MUX_`.